### ISSUE TITLE
Volvo Migration: Configure Repo for Prod Board

### ISSUE THREAD

**Issue by mollyaljenkins** [hlxsites/vg-volvotrucks-us#709]
Related to the Volvo Repo Migration #691 

As an Agile team, we want to be able to access all Volvo issues from the main production board on GitHub, so that all issues for the Volvo Group can be visible in one place. 

**AC:** 
- [ ] The new aem-sites repo for Volvo Trucks is configured so that all Volvo Trucks issues can be visible and worked on directly from the main VolvoGroup - Franklin Sites (PRODUCTION) Agile production board

**Notes:**
- This issue is pending the same solution for aem-sites/vg-roadchoice-com and is to be implemented once a solution has been found via @dkuntze 
- Currently it is not possible to transfer issues from one board to another as the repos have different owners

**Comment by mollyaljenkins** [comment]
Blocked as waiting on solution to migrating issues between board with different owners. @dkuntze

**Comment by mollyaljenkins** [comment]
This ticket is closed as won't do, a new production board will need to be created for the AEM repos.
